ow_power_system/src/PowerCSV.cpp

```diff
@@ -0,0 +1,57 @@
+//PowerCSV.cpp
+
+/*The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+Research and Simulation can be found in README.md in the root directory of
+this repository.*/
+
+#include "PowerCSV.h"
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+
+/* Function to load pre-generated values as a vector of vectors of doubles
+ * Returns std::vector<std::vector<std::double>>
+ */
+std::vector<std::vector<double>> powerCSV::loadCSV(const std::string& filename){
+  ifstream power_csv(filename);
+  if (power_csv.fail()) {
+    cout << "Unable to open data file" << endl;
+  }
+  vector<vector<double>> values;
```

Comment:
I'm not sure how 'vector' works here without the 'std::' prefix, like you use everywhere above, plus in the header file.  I strongly suggest putting a 'using std::vector' at the top of this file for clarity, and drop the 'std::' from vector everywhere for consistency.